lib/screens/login.dart

```diff
@@ -1,10 +1,52 @@
 import 'package:flutter/material.dart';
 import 'package:math_go/screens/map_view.dart';
 import 'package:flutter_native_splash/flutter_native_splash.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:math_go/screens/tutorial.dart';
 
-class LoginScreen extends StatelessWidget {
+class LoginScreen extends StatefulWidget {
   const LoginScreen({Key? key}) : super(key: key);
 
+  @override
+  State<LoginScreen> createState() => _LoginScreenState();
+}
+
+class _LoginScreenState extends State<LoginScreen> {
+  //Checks if a user is currently logged in.
+  bool _isDisabled() {
+    if (FirebaseAuth.instance.currentUser == null) {
```

Comment:
You can simplify this to just say return FirebaseAuth.instance.currentUser == null;

Reply:
Simplified the code as you suggested.  Thanks.